packages/variable-editor/src/components/variables/master/OverwriteDialog.tsx

```diff
@@ -78,7 +59,31 @@ export const OverwriteDialog = ({ knownVariables, table, variables, setVariables
     }
   };
 
-  const VariableBrowser = ({ applyFn }: { applyFn: (node?: ProjectVarNode) => void }) => {
+  const VariableBrowser = ({ applyFn, context }: { applyFn: (node?: ProjectVarNode) => void; context: DataContext }) => {
+    const client = useClient();
+
+    const queryKeys = useMemo(() => {
+      return {

```

Comment:
if you do nothing else than return something, you can also use the sorthand syntax for it:

```tsx
const queryKeys = useMemo(() => ({ knownVariables: () => genQueryKey('meta/knownVariables', context) }), [context]);
```